### PR TITLE
Indicate which DNS providers support DoT over port 443

### PR DESCRIPTION
**Pull request by nitrohorse**
## Description

This PR adds a tooltip to the providers who support DoT over port 443 in addition to the expected port 853. This is an important indication because like DoH, DoT/443 makes eavesdropping more difficult for third parties and provides the benefit of not easily being blocked by firewalls compared to 853.

Resolves: #none

#### Check List 

- [x] I have read and understand [CONTRIBUTING.md](https://github.com/privacytoolsIO/privacytools.io/blob/master/.github/CONTRIBUTING.md).

- ~~[ ] I have listed the source code for this project in [source_code.md](https://github.com/privacytoolsIO/privacytools.io/blob/master/source_code.md).~~

- ~~[ ] This project is [free/libre software](https://www.wikipedia.org/wiki/Free_software).~~

- ~~[ ] This project has an [associated discussion](https://github.com/privacytoolsIO/privacytools.io/issues).~~ (via Wire chat)

Code Repository (if applicable): N/A

### CODE REVIEW

**Comment by nitrohorse** [comment]
Hmm, these are good points :thinking: 

> I think the colour is too light or difficult to spot

Yeah that makes sense, what if I also bold the text instead of relying on just a color change? Will push an update to try it.

> 
> I wish the sorting preferred DoT in port 443 over DoT in port 853 due to below
> 

Okay, I'll give those three providers' protocol field a custom value so they're grouped together while everything else sorts by length? I'll push an update for this.

> Could the terms be updated to clarify this? I am thinking that DoT should mention "...which is often blocked by restrictive firewalls, while 443 generally works everywhere"?

Good callout; will do :+1: 
>       
> I started wondering about DPI that can distinguish SSH from https, but can it distinguish TLS encrypted traffic from other https traffic?

That's a good question; I'll do some digging into this.

**Comment by JonahAragon** [comment]
I wonder if green is the best color choice, does it imply that configuration is more secure than others? I'd probably prefer using `text-info` or maybe `text-primary`. Or even just black with bold text.

**Comment by nitrohorse** [comment]
> I wonder if green is the best color choice, does it imply that configuration is more secure than others? I'd probably prefer using `text-info` or maybe `text-primary`. Or even just black with bold text.

I've removed the color; you're right; DoT/443 isn't necessarily more secure from my understanding, but doesn't stand out as much. In other words, it's more difficult to eavesdrop on and block.

**Comment by Mikaela** [comment]
> In other words, it's more difficult to eavesdrop on and block.
> 
> @Mikaela this makes me think our description for DoT/443 in the terms could mention that eavesdropping is more difficult. Currently it says:

What do you mean by eavesdropping? The queries are encrypted from you to the DNS server and no one else can read them.

**Comment by nitrohorse** [comment]
@Mikaela ah sorry, by eavesdropping, I mean DoT/443 could potentially be more difficult for your ISP/3rd parties to _analyze_ and _detect_ than DoT/853, yeah?

**Comment by Mikaela** [comment]
> by eavesdropping, I mean DoT/443 could potentially be more difficult for your ISP/3rd parties to analyze and detect than DoT/853, yeah?

Possibly yes, but I am not comfortable making that claim personally until someone wiser tells me so

**Comment by Mikaela** [comment]
As I went jabbering about this [on the forum](https://forum.privacytools.io/t/dns-over-https-doh-vs-dns-over-tls-dot-where-do-you-stand/1433?u=mikaela) could we have a review and merge? :)

File: _includes/sections/dns.html [hunk 1]
```diff
@@ -150,7 +150,7 @@ <h1 id="icanndns" class="anchor"><a href="#icanndns"><i class="fas fa-link ancho
         </td>
         <td>Hobby Project</td>
         <td>No</td>
-        <td>DoH, DoT, DNSCrypt</td>
+        <td data-value="dot/443">DoH, <span data-toggle="tooltip" data-placement="bottom" data-original-title="Supports port 443 in addition to 853"><strong>DoT</strong></span>, DNSCrypt</td>
```
Comment (dawidpotocki) [inline _includes/sections/dns.html]: What about adding question mark next do DoT?
It is hard to tell that there is hidden info.

**Pull request merged**